Recrutify/Recrutify.Host/Controllers/SchedulesController.cs

```diff
@@ -35,6 +35,14 @@ public Task<ScheduleDTO> GetByDatePeriodForCurrentUserAsync([FromQuery] DateTime
             return _scheduleService.GetByDatePeriodForCurrentUserAsync(date, daysNum);
         }
 
+        [HttpPut("bulk/appoint_cancel_interviews")]
+        public async Task<ActionResult> BulkAppointOrCancelInterviewsAsync([FromBody] BulkAppointInterviewsDTO bulkAppointInterviewsDTO, [FromQuery, Required] Guid projectId)
+        {
+            await _scheduleService.BulkAppointOrCancelInterviewsAsync(bulkAppointInterviewsDTO.InterviewAppointments, bulkAppointInterviewsDTO.ProjectId);
```

Comment:
You have project id in bulk dto, not necessary to pass it explicitly, in query is only for permission check